tools/mmp/driver.cs

```diff
@@ -902,10 +902,9 @@ static void CopyMonoNative ()
 				// copy to temp directory and lipo there to avoid touching the final dest file if it's up to date
 				var temp_dest = Path.Combine (App.Cache.Location, "libmono-native.dylib");
 
-				if (Application.UpdateFile (src, temp_dest)) {
+				if (Application.UpdateFile (src, temp_dest))
 					LipoLibrary (name, temp_dest);
-					Application.CopyFile (temp_dest, dest);
-				}
+				Application.UpdateFile (temp_dest, dest);
```

Comment:
This seems fine, but why was it changed here?

Reply:
See this commit by @spouliot: https://github.com/xamarin/xamarin-macios/commit/224593b46df18a514afa0a5281258f79988c8f02